examples/workers/workers.bal

```diff
@@ -19,6 +19,10 @@ public function main() {
         io:println("sum of first ", n, " positive numbers = ", sum);
     }
 
+    // By default, workers run on the same physical thread of the parent strand.
+    // By adding @concurrent annotation, this worker's strand is allowed to run
```

Comment:
```suggestion
    // By adding the `@concurrent` annotation, this worker's strand is allowed to run
```

Reply:
This doesn't have `{}` for the annotation btw. Shouldn't this be consistent with https://github.com/ballerina-platform/ballerina-lang/pull/20350/files#diff-8ab9d08acec1364f962b1a776e9cec9fR29?